src/test/py/bazel/bazel_windows_cpp_test.py

```diff
@@ -384,7 +384,8 @@ def testDLLIsCopiedFromExternalRepo(self):
     # Test if A.dll is copied to the directory of main.exe
     main_bin = os.path.join(bazel_bin, 'main.exe')
     self.assertTrue(os.path.exists(main_bin))
-    self.assertTrue(os.path.exists(os.path.join(bazel_bin, 'A_729d833d.dll')))
+    self.assertEqual(
+      len(glob.glob(os.path.join(bazel_bin, 'A_*.dll'))), 1)
```

Comment:
@meteorcloudy Is this okay? I had no idea how to update the hash.

Reply:
Yeah, this is definitely better, no reason to enforce the exact hash.